### PR TITLE
Adding survivor merge for sv vcfs

### PR DESCRIPTION
**Pull request by apaul7**
this will allow us to merge the results from different sv callers together into a single vcf. And then annotate them with a population frequency of the sv calls.

I have it set up as a separate sub workflow to make it easier for people to run this manually. 

Open to suggestions on naming of files and input variables.

### CODE REVIEW

File: definitions/tools/add_sv_population_frequency.cwl [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    - class: DockerRequirement
+      dockerPull: "halllab/svtools:v0.4.0-2dc7f91" 
+    - class: ResourceRequirement
+      ramMin: 5000
+      coresMin: 1
+
+inputs:
+    vcf:
+        type: File
+        doc: "vcf to add populationa allele frequence to"
```
Comment (tmooney) [inline definitions/tools/add_sv_population_frequency.cwl]:
```suggestion
        doc: "vcf to add population allele frequencies to"
```

File: definitions/tools/survivor.cwl [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    same_strand:
+        type: int
+        inputBinding:
+            position: 5
+        doc: "Require merged SVs to be on the same strand, 1=yes"
+    estimate_sv_distance:
+        type: int
+        inputBinding:
+            position: 6
+        doc: "Estimate distance based on the sixe of SV, 1=yes"
```
Comment (tmooney) [inline definitions/tools/survivor.cwl]:
```suggestion
        doc: "Estimate distance based on the size of SV, 1=yes"
```

File: definitions/tools/survivor.cwl [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    minimum_sv_calls:
+        type: int
+        inputBinding:
+            position: 3
+        doc: "Minimum number of sv calls needed to be merged"
+    same_type:
+        type: int
+        inputBinding:
+            position: 4
+        doc: "Require merged SVs to be of the same type, 1=yes"
```
Comment (tmooney) [inline definitions/tools/survivor.cwl]: Is anything else `no`?

Comment (apaul7) [reply definitions/tools/survivor.cwl]: it's 1=yes, 0=no. updated documentation to state that. I was going to use the boolean input type but thought that was for flag inputs. this has to have something as all inputs are positional

Comment (tmooney) [reply definitions/tools/survivor.cwl]: Yeah, the boolean type is mostly for flag inputs.  You could use JavaScript to convert it to a `1` or a `0`, or possibly define an enumerated type here that accepts only `0` and `1`.

**Comment by apaul7** [comment]
will merge this and make a new pr to add it to the single sample sv workflow

**Pull request merged**